go.mod

```diff
@@ -0,0 +1,8 @@
+module go.uber.org/ratelimit
+
+go 1.14
+
+require (
+	github.com/stretchr/testify v1.6.1
+	go.uber.org/atomic v1.6.0
```

Comment:
we just cut a new release. mind upgrading to 1.7.0?

Reply:
sure.